rest/taskana-rest-spring/src/main/java/pro/taskana/task/rest/assembler/TaskCommentRepresentationModelAssembler.java

```diff
@@ -70,9 +68,9 @@ public TaskanaPagedModelKeys getProperty() {
   }
 
   @Override
-  @PageLinks(Mapping.URL_TASK_COMMENTS)
   public TaskanaPagedModel<TaskCommentRepresentationModel> toPageModel(
       Iterable<TaskComment> taskComments, PageMetadata pageMetadata) {
-    return TaskanaPagingAssembler.super.toPageModel(taskComments, pageMetadata);
+    return addLinksToPageResource(
+        TaskanaPagingAssembler.super.toPageModel(taskComments, pageMetadata), pageMetadata);
```

Comment:
Why do we add the metadata twice and do not simply get it from the model?

Reply:
You're right! Thank you!